pandera/schemas.py

```diff
@@ -230,9 +231,9 @@ def _set_column_handler(column, column_name):
         }
 
     @property
-    def dtype(self) -> Dict[str, str]:
+    def dtypes(self) -> Dict[str, str]:
```

Comment:
note here that we should keep track of the breaking changes in the documentation.

we also don't have an official change log, which we should really start 😅

Reply:
I've listed breaking changes in the top post. 

Type annotations is wrong, now it returns a  `Dict[str, Datatype]` as mentioned in the breaking changes.

Reply:
we should also update the documentation to reflect this:
https://pandera.readthedocs.io/en/stable/dataframe_schemas.html#get-pandas-datatypes

I think this part of the docs pre-dated the `coerce` keyword, so the code example doesn't really make sense anymore
```python
schema = pa.DataFrameSchema(
    columns={
      "column1": pa.Column(pa.Int),
      "column2": pa.Column(pa.Category),
      "column3": pa.Column(pa.Bool)
    },
    coerce=True,
)

df = pd.DataFrame.from_dict(
  {
      "a": {"column1": 1, "column2": "valueA", "column3": True},
      "b": {"column1": 1, "column2": "valueB", "column3": True},
  },
  orient="index"
).astype(schema.dtype).sort_index(axis=1)

# This can just be
schema(df)
```

Reply:
> I think the dtypes and get_dtypes attributes might still be useful, if not to just conveniently inspect the (pandera engine) dtypes

Totally !